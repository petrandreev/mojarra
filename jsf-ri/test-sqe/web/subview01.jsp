<!--
 The contents of this file are subject to the terms
 of the Common Development and Distribution License
 (the License). You may not use this file except in
 compliance with the License.
 
 You can obtain a copy of the License at
 https://javaserverfaces.dev.java.net/CDDL.html or
 legal/CDDLv1.0.txt. 
 See the License for the specific language governing
 permission and limitations under the License.
 
 When distributing Covered Code, include this CDDL
 Header Notice in each file and include the License file
 at legal/CDDLv1.0.txt.    
 If applicable, add the following below the CDDL Header,
 with the fields enclosed by brackets [] replaced by
 your own identifying information:
 "Portions Copyrighted [year] [name of copyright owner]"
 
 [Name of File] [ver.__] [Date]
 
 Copyright 2005 Sun Microsystems Inc. All Rights Reserved
-->
<%@ page contentType="text/html" language="java" %>
<%@ taglib prefix="c" uri="http://java.sun.com/jstl/core" %>
<%@ taglib prefix="f" uri="http://java.sun.com/jsf/core" %>
<%@ taglib prefix="h" uri="http://java.sun.com/jsf/html" %>

<f:view>
<html>
<head>
<title>subview01</title>
</head>
<body>
<h:outputText value="Begin test <c:import> with subview tag in imported page"/>
<c:import url="foo01.jsp"/>
<h:outputText value="subview01"/>
<c:import url="bar01.jsp"/>
<h:outputText value="End test <c:import> with subview tag in imported page"/>
</body>
</html>
</f:view>
