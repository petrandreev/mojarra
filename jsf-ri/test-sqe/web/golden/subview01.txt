<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->






<html>
<head>
<title>subview01</title>
</head>
<body>
Begin test &lt;c:import&gt; with subview tag in imported pagefoo01subview01bar01End test &lt;c:import&gt; with subview tag in imported page
</body>
</html>

